Doesn't pass the pretiter lint

# Bug
Doesn't pass the pretiter lint

```sh
  49:77  error  Delete `·this`  prettier/prettier
  50:16  error  Insert `·this`  prettier/prettier

✖ 2 problems (2 errors, 0 warnings)
  2 errors and 0 warnings potentially fixable with the `--fix` option.
```

## Environment info
React native info output:

```bash
System:
    OS: Linux 5.3 Ubuntu 19.10 (Eoan Ermine)
    CPU: (4) x64 Intel(R) Core(TM) i5-4690K CPU @ 3.50GHz
    Memory: 971.49 MB / 31.31 GB
    Shell: 5.0.3 - /bin/bash
  Binaries:
    Node: 13.5.0 - ~/.nvm/versions/node/v13.5.0/bin/node
    Yarn: 1.12.3 - /opt/node-v11.0.0-linux-x64/bin/yarn
    npm: 6.13.4 - ~/.nvm/versions/node/v13.5.0/bin/npm
  npmPackages:
    react: 16.9.0 => 16.9.0 
    react-native: 0.61.5 => 0.61.5
```

Library version: 9eba4e6e48ce323b52fb9660db691c5c7750edad

## Steps To Reproduce

<!--
 Issues without reproduction steps or code are likely to stall.
-->

1. npm run lint
2.
...

Describe what you expected to happen:

1. Not fail
2.

## Reproducible sample code

<!--
 Please add to your issue a repro, a fresh codebase with the minimal changes so that the bug can be tested in isolation
-->

--- a/template/App.tsx
+++ b/template/App.tsx
@@ -46,8 +46,8 @@ const App = () => {
             <View style={styles.sectionContainer}>
               <Text style={styles.sectionTitle}>Step One</Text>
               <Text style={styles.sectionDescription}>
-                Edit <Text style={styles.highlight}>App.tsx</Text> to change this
-                screen and then come back to see your edits.
+                Edit <Text style={styles.highlight}>App.tsx</Text> to change
+                this screen and then come back to see your edits.
               </Text>
             </View>
             <View style={styles.sectionContainer}>

